metron-platform/metron-integration-test/src/main/config/zookeeper/global.json

```diff
@@ -7,10 +7,12 @@
   "solr.collection": "metron",
   "solr.numShards": 1,
   "solr.replicationFactor": 1,
+  "flushIntervalInMs": "20000",
+  "flushOnTimeout": "false",
```

Comment:
Do we need both parameters?  If we have a flush interval set, we should infer that we want to do a time-based flush, right?